expense-rescript/src/components/NewTransaction/NewTransaction.css

```diff
@@ -0,0 +1,34 @@
+.new-transaction {
+  padding: 1rem;
+}
+
+.error-label {
+  color: red;
+  text-align: center;
+}
+
+.new-transaction input {
+  width: 100%;
+  border: 0.1rem solid #ccc;
+  box-sizing: border-box;
+  padding: 0.8rem 0.8rem;
+  box-shadow: inset;
+}
+
+.new-transaction label {
+  font-weight: bold;
+  display: block;
+  margin: 0.4rem 0 0.4rem 0;
+}
+
+.new-transaction button {
+  width: 100%;
+  background-color: #8c1aff;
+  border: none;
+  color: white;
+  padding: 1rem;
+  text-align: center;
+  display: inline-block;
```

Comment:
Check if these are providing any additional value

Reply:
I removed it, wasn't providing any additional value.